src/PackageDetailsViewExtension/PackageDetailsViewModel.cs

```diff
@@ -201,15 +200,29 @@ PackageManagerSearchElement packageManagerSearchElement
             PackageDetailsViewExtension = packageDetailsViewExtension;
             License = packageManagerSearchElement.Header.license;
 
+            packageLoader.PackageAdded += PackageLoaderOnPackageAdded;
```

Comment:
Would you make sure we unsubscribe from this?

Reply:
Added a dispose method, which disposes along with the extension.